google/cloud/datastore/client.py

```diff
@@ -77,7 +78,7 @@ def _extended_lookup(connection, project, key_pbs,
     :type project: string
     :param project: The project to make the request for.
 
-    :type key_pbs: list of :class:`google.cloud.datastore._generated.entity_pb2.Key`
+    :type key_pbs: list of :class:`.datastore._generated.entity_pb2.Key`
```

Comment:
See above. The `.` isn't what it originally seems (I also thought it was a relative path).